.github/CODEOWNERS

```diff
@@ -1188,6 +1189,7 @@ x-pack/test_serverless/**/test_suites/observability/ai_assistant @elastic/obs-ai
 # Observability UX management team
 /x-pack/packages/observability/alert_details @elastic/obs-ux-management-team
 /x-pack/test/observability_functional @elastic/obs-ux-management-team
+/x-pack/test/functional/es_archives/cases/default @elastic/obs-ux-management-team
```

Comment:
I think this should be owned by the `@elastic/response-ops` even though it is used only in `x-pack/test/observability_functional/apps/observability/feature_controls/observability_security.ts`. @elastic/obs-ux-management-team wdyt?

Reply:
thanks for chiming in Christos!

Reply:
Agree!

Reply:
I agree 😆 !